Meadowlark-js/services/meadowlark-aws-lambda/serverless.yml

```diff
@@ -50,11 +29,10 @@ provider:
     minimumCompressionSize: 1024 # Enable gzip compression for responses > 1 KB
 
   environment:
-    AWS_NODEJS_CONNECTION_REUSE_ENABLED: 1 # Reuse TCP connections - important for DynamoDB performance
+    # AWS_NODEJS_CONNECTION_REUSE_ENABLED: 1   # Explore this during performance testing.
```

Comment:
The AWS documentation that mentions this setting explicitly states that it is useful for short-lived interactions with DynamoDB. Not clear if it might be useful in other settings. I thought it would be good to leave it in place as a reminder for us to explore during performance testing. Open to removing it though.

Reply:
Makes sense